src/java.base/share/classes/java/lang/Double.java

```diff
@@ -143,6 +143,55 @@
  * elements of a {@link java.util.SortedSet SortedSet} or as keys of a
  * {@link java.util.SortedMap SortedMap}.
  *
+ * <p>While {@code ==} is not an equivalence relation, several useful
+ * equivalence relations can be defined over floating-point
+ * values. Comparing numerical equality to various kinds of
+ * equivalence:
```

Comment:
The overall discussion is good, and the inclusion of numerical equality is good, but now this introductory paragraph is invalid. It starts off:

> While `==` is not an equivalence relation,

OK, this continues the discussion, picking up from the assertion several paragraphs above that `==` is not an equivalence relation. Then,

> several useful equivalence relations can be defined over floating-point values. Comparing numerical equality to various kinds of equivalence:

It sounds like what follows will be a list of equivalence relations that will be contrasted with numerical equality, yet the very first example is numerical equality and is _not_ an equivalence relation.

I think this just needs to be reworded to be more inclusive. Maybe something like, "the following is a comparison of various relations among floating-point values."